src/components/Frame/components/Loading/Loading.tsx

```diff
@@ -20,11 +19,6 @@ export default class Loading extends React.Component<Props, State> {
     animation: null,
   };
 
-  private ariaValuenow = debounce(() => {
```

Comment:
Debounce doesn't return a value anymore. State changes are also triggered from RAF which runs every ~16ms which is longer than the debounce time.

Reply:
We should still probably update debounce to return the return value. This was an oversight on my part because all of our uses at the time were side effects.